examples/outcomes/1/outcomes-lb.json

```diff
@@ -0,0 +1,9 @@
+{
+  "project_id": 1780659,
```

Comment:
I know it's not exactly customer info but let's be more paranoid and please replace these all with `project_id`: 1

Reply:
Same goes for org ids